domains/toadvillebay.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "notghostvista",
+        "email": "toadvillebay@gmail.com"
+    },
+    "record": {
+        "RECORDTYPE": "CNAME"
```

This is not how records work
```suggestion
        "CNAME": "your.domain.com"
```